src/Microsoft.Health.Fhir.SqlServer/Features/Schema/Migrations/1.sql

```diff
@@ -1250,6 +1250,33 @@ WITH (DATA_COMPRESSION = PAGE)
 
 GO
 
+/*************************************************************
+    Export Job
+**************************************************************/
+CREATE TABLE dbo.ExportJob
```

Comment:
Is the plan to move these additions to a `2.sql` before merging the feature branch to master?

Reply:
Yep, I plan to complete [#AB71190](https://microsofthealth.visualstudio.com/Health/_workitems/edit/71190) before merging.